package.json

```diff
@@ -34,9 +34,9 @@
     "stage:venia": "yarn venia run start",
     "stats:venia": "yarn venia run build:analyze && yarn venia run stats",
     "storybook:venia": "yarn venia run storybook",
-    "test": "jest",
-    "test:ci": "jest --no-cache -i --json --outputFile=test-results.json",
-    "test:debug": "node --inspect-brk node_modules/.bin/jest -i --no-cache",
+    "test": "node --experimental-worker node_modules/.bin/jest",
+    "test:ci": "node --experimental-worker node_modules/.bin/jest --no-cache -i --json --outputFile=test-results.json",
```

Comment:
![image](https://user-images.githubusercontent.com/35203638/68709729-e1ff7700-055b-11ea-8c16-6ee631bc57b6.png)


```suggestion
    "test:ci": "node --experimental-worker node_modules/.bin/jest --no-cache --json --outputFile=test-results.json",
```

If we remove the `-i` flag from `test:ci` jest does not complain about `MessagePort` open handles. @zetlen @jimbo let me know how to solve this?

Reply:
Let's keep this open until you really resolve the whole issue. Seems like removing the flag still causes problems. I suggested in slack -- I think we just use `-i` and let it warn (only if we trash the instances after running tests).